src/pages/Home.tsx

```diff
@@ -30,20 +30,26 @@ const Home: React.FC = () => {
         has_more: false
     });
     const [searchQuery, setSearchQuery] = useState<string>("");
-    const [sortOrder, setSortOrder] = useState<string>("desc");
+    const [sortOrder, setSortOrder] = useState<string>(() => {
+        return localStorage.getItem("defaultSort") || "desc";
+    });
+    const [category, setCategory] = useState<string>(() => {
+        return localStorage.getItem("defaultSentiment") || "";
+    });
     const [darkMode, setDarkMode] = useState<boolean>(() => {
         return localStorage.getItem("darkMode") === "true";
     });
     const [isLoading, setIsLoading] = useState<boolean>(false);
 
     // Fetch articles with pagination and sorting
-    const loadArticles = async (offset = 0, keyword = "", sort = sortOrder) => {
+    const loadArticles = async (offset = 0, keyword = "", sort = sortOrder, selectedCategory = category) => {
         setIsLoading(true);
         try {
             const response = await fetchArticles({
                 offset,
                 keyword,
-                sort: sort as 'asc' | 'desc'
+                sort: sort as 'asc' | 'desc',
```

Comment:
Changed the sort strings to asc and desc since it's more appropriate for the API.